R/plot.R

```diff
@@ -60,6 +60,14 @@ plot.bayestestR_eti <- function(x, ...) {
   NextMethod()
 }
 
+#' @export
+plot.bayestestR_eti <- function(x, ...) {
+  # if (!requireNamespace("see", quietly = TRUE)) {
+  #   stop("Package 'see' needed to plot credible intervals. Please install it by running `install.packages('see')`.")
+  # }
+  stop("No plotting method for SIs (yet)..")
+}
+
```

Comment:
I would already activate plotting, the implementation in _see_ should be fairly easy, or not?

Reply:
I'm not in the right head space for this just yet (still with a bit of the flu)... Hope to get to the plotting for the next PR from dev.

Also, Daniel, can you help with the last issue in #266 ? That of `ci(...,method = "SI")`?